### DISCUSSION TITLE
Override Existing Errors

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-discussions#144] [Extension Development QnA] [answered]
![Screenshot from 2022-09-28 14-59-24](https://user-images.githubusercontent.com/98210802/192744203-cfc2a902-4992-445f-9fb8-705e27e3efcb.png)
How Can I override/remove that error using an extension.

**Comment by r3m0t** [comment, marked as answer]
Use a [typescriptServerPlugin ](https://code.visualstudio.com/api/references/contribution-points#contributes.typescriptServerPlugins) to amend the behaviour of tsserver before it generates that error

**Reply by ghost** [reply]
Thanks 😌
